lightning/src/chain/chainmonitor.rs

```diff
@@ -40,10 +41,12 @@ use ln::channelmanager::ChannelDetails;
 use prelude::*;
 use sync::{RwLock, RwLockReadGuard, Mutex};
 use core::ops::Deref;
+use core::sync::atomic::{AtomicBool, Ordering};
 
 #[derive(Clone, Copy, Hash, PartialEq, Eq)]
 pub(crate) enum MonitorUpdate {
 	MonitorUpdateId(u64),
+	SyncPersistId(u64),
```

Comment:
Is the u64 here used anywhere?

Reply:
Only in the `#[derive(Hash, PartialEq)]` which is used for removing the same element in the HashSet.